app/server-utils/stampy.ts

```diff
@@ -174,7 +175,7 @@ const extractLink = (markdown: string) => markdown?.replace(/^.*\(|\)/g, '')
 const convertToQuestion = (title: string, v: CodaRow['values']): Question => ({
   title,
   pageid: extractText(v['UI ID']),
-  text: v['Rich Text'] ? md.render(extractText(v['Rich Text'])) : null,
+  text: v['Rich Text'] ? urlToIframe(md.render(extractText(v['Rich Text']))) : null,
```

Comment:
this doesn't look right - the previous version should be `text: renderText(v['Rich Text']),` - are you sure you're up to date?

Reply:
You were right; it wasn't upto date. I had to merge in latest `master` branch

Reply:
Fixed with https://github.com/StampyAI/stampy-ui/pull/148/commits/3af60d3f7db40d6332f1273b9476129ae9c99589